src/components/ha-push-notifications-toggle.js

```diff
@@ -93,7 +106,25 @@ class HaPushNotificationsToggle extends EventsMixin(PolymerElement) {
         return;
       }
 
-      sub = await reg.pushManager.subscribe({ userVisibleOnly: true });
+      let applicationServerKey = null;
+      try {
+        const res = await this.hass.callApi(
```

Comment:
Please:
 - use `hass.callWS`
 - Extract the call to a new function in `src/data/notify_html5.ts`